packages/cli/src/settings.ts

```diff
@@ -6,9 +6,14 @@ import { z } from 'zod';
 
 import { ProjectLanguage } from './project/project';
 
+const enablePython = process.env.ENABLE_EXPERIMENTAL_PYTHON_OPTIONS === 'true' || process.env.ENABLE_EXPERIMENTAL_PYTHON_OPTIONS === '1';
+const languageEnum = enablePython
```

Comment:
nit: since your not using this variable in more than one place an inline assignment would be cleaner